.github/workflows/twas-baseBuild.yml

```diff
@@ -193,13 +216,32 @@ jobs:
           blobStorageEndpoint=$( az storage account show -n ${{ env.vhdStorageAccountName }} -g ${{ env.testResourceGroup }} -o json | jq -r '.primaryEndpoints.blob' )
           osDiskSasUrl=${blobStorageEndpoint}vhds/${{ env.vmName }}.vhd?$sasToken
           dataDiskSasUrl=${blobStorageEndpoint}vhds/${{ env.vmName }}datadisk1.vhd?$sasToken
-          
+
           echo "osDiskSasUrl: ${osDiskSasUrl}, dataDiskSasUrl: ${dataDiskSasUrl}" > sas-url-twasbase.txt
+          echo "##[set-output name=osDiskSasUrl;]${osDiskSasUrl}"
+          echo "##[set-output name=dataDiskSasUrl;]${dataDiskSasUrl}"
       - name: Upload SAS url
         uses: actions/upload-artifact@v3
         with:
           name: sasurl-twasbase
           path: sas-url-twasbase.txt
+      - name: Update offer sas url and version
+        uses: microsoft/microsoft-partner-center-github-action@v3
+        if: ${{ github.repository_owner == 'WASdev' }}
+        with:
+          clientId: ${{ env.clientId }}
+          dataDiskSasUrl: ${{steps.sas_url.outputs.dataDiskSasUrl}}
+          imageType: ${{ env.imageType }}
+          imageVersionNumber: ${{ steps.setup-env-variables-based-on-dispatch-event.outputs.imageVersionNumber }}
+          offerId: ${{ env.offerId }}
+          offerType: ${{ env.offerType }}
+          operatingSystemFamily: ${{ env.operatingSystemFamily }}
+          operatingSystemType: ${{ env.operatingSystemType }}
+          osDiskSasUrl: ${{steps.sas_url.outputs.osDiskSasUrl}}
+          planId: ${{ env.planId }}
+          secretValue: ${{ env.secretValue }}
+          tenantId: ${{ env.tenantId }}
+          verbose: "true"
```

Comment:
Same as https://github.com/WASdev/azure.websphere-traditional.image/pull/77/files#r1195729282.

Reply:
Fixed in af9f670.